docs/rst/tutorials/core/qos/initial_peers/initial_peers.rst

```diff
@@ -0,0 +1,219 @@
+
+For example:
+
+.. code-block:: bash
+
+    cp <path_to_xml_file> DEFAULT_FASTRTPS_PROFILES.xml
+    export RMW_FASTRTPS_USE_QOS_FROM_XML=1
+
+.. important::
+
+    Repeat the same procedure in the other opened Vulcanexus docker instance.
+
+.. _initial_peers_example:
+
+Example
+-------
+
+Once both Vulcanexus docker environments have been configured, run a demo application in each instance:
+
+.. code-block:: bash
+
+      # Talker
+      ros2 run demo_nodes_cpp talker
+
+      # Listener
+      ros2 run demo_nodes_cpp listener
+
+The Vulcanexus docker instances would discover each other via multicast, then one of them would let the other know its address and port through it would establish any communication, and they will connect.
+
+.. _initial_peers_unicast_example:
+
+Unicast Example
+^^^^^^^^^^^^^^^
+
+If unicast is required instead of multicast, the following profile should be updated in the existing XML profile:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+        <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+            <participant profile_name="initial_peers_example_profile" is_default_profile="true">
+                <rtps>
+                    <builtin>
+                        <initialPeersList>
+                            <locator>
+                                <udpv4>
+                                    <address>172.17.0.3</address> <!-- "keen_john"  IP address -->
+                                    <port>7777</port>             <!-- "keen_john"  port       -->
+                                </udpv4>
+                            </locator>
+                        </initialPeersList>
+                        <metatrafficUnicastLocatorList>
+                            <locator>
+                                <udpv4>
+                                    <address>172.17.0.2</address> <!-- "eager_pike" IP address -->
+                                    <port>7666</port>             <!-- "eager_pike" port       -->
+                                </udpv4>
+                            </locator>
+                        </metatrafficUnicastLocatorList>
+                    </builtin>
+                </rtps>
+            </participant>
+        </profiles>
+
+In this case, the port `7777` is not longer the default port, it is a new custom port to reach the peer address.
+Furthermore, the meta-traffic unicast locator has been set with the Vulcanexus docker instance IP address (own address), and the port it would be communicating through.
+
+Create a new XML file and complete it with the following example in the non-existing xml file docker instance:
+
+.. literalinclude:: /resources/tutorials/core/qos/initial_peers/initial_peers_unicast_config.xml
+    :language: xml
+
+Finally, the addresses and ports have been swapped compared to the previous XML file, in order to match IP addresses and ports in which they are set to use.
```

Comment:
Remove or rephrase as it's difficult to understand what you want to say with this swap.